build.gradle.kts

```diff
@@ -27,7 +27,12 @@ plugins {
   id("com.github.gmazzo.buildconfig") version "2.1.0" apply false
 }
 
-subprojects {
+apply(plugin = "com.vanniktech.maven.publish.base")
+
+allprojects {
+  group = project.property("GROUP") as String
+  version = project.property("VERSION_NAME") as String
```

Comment:
Let's just inline these two and update the `RELEASING.md`?

Reply:
Hell yes. Done.